Libraries/Components/Touchable/TouchableBounce.js

```diff
@@ -11,7 +11,7 @@
 
 const Animated = require('Animated');
 const DeprecatedViewPropTypes = require('DeprecatedViewPropTypes');
-const EdgeInsetsPropType = require('EdgeInsetsPropType');
+const EdgeInsetsPropType = require('DeprecatedEdgeInsetsPropType');
```

Comment:
Can you rename the variable to match the require?

```
const DeprecatedEdgeInsetsPropType = require('DeprecatedEdgeInsetsPropType');
```

Reply:
Sure, also should I changed in react-native-implementation 
```
  // Prop Types
  get ColorPropType() {
    return require('ColorPropType');
  },
  get EdgeInsetsPropType() {
    return require('EdgeInsetsPropType');
  },
```
to
```
  // Prop Types
  get ColorPropType() {
    return require('ColorPropType');
  },
  get EdgeInsetsPropType() {
    return require('DeprecatedEdgeInsetsPropType');
  },
```
?